### DISCUSSION TITLE
Bachelor Thesis, Offshore windfarm energy analysis

### DISCUSSION THREAD

**Discussion by r1nov8** [NatLabRockies/floris#807] [Q&A] [answered]
Hi,

I am an energy engineering student doing my bachelor thesis,. I am quite new to python, and was wondering if anybody could guide me in the right direction. or even tell me if this is possible. 

My thesis is about analysing the energy loss due to wake effects on a wind farm in a specific area off the coast of Norway, the area is called Vestavind F. the mission is to compare 3 different turbine types. 5MW,10MW and 15MW. but the last one is most important. I want a layout of 10*5 turbine layout. and i want to use NORA3 winddata i have obtained to analyse the AEP with wake loss, and what it would be with an optimized wind park layout. 

Is this possible?
If so, any idea where i should start. 

i have managed to make my own windrose in the floris toolbox using the NORA3 data. But really dont know where to go from there.

thank you in advanced for any help.
![Figure_1](https://github.com/NREL/floris/assets/156081336/ddb389e2-2c6c-4ba4-b9c3-2c6352fa5056)

**Comment by paulf81** [comment, marked as answer]
hi @r1nov8, thank you for your question, this is all very possible!   The best place to start would be with the examples folder and scripts.  I think going through those you will find all the elements you need to complete this work.  For a few starting suggestions, have a look at examples 07/08, this computes AEP for a farm with and without wake losses.  In terms of optimized layout, look at example 15.  Good luck!

**Reply by r1nov8** [reply]
Thank you for the good suggestions,

A quick question, have managed to run the 07/08 examples with my wind data, but i am having a little trouble with implementing the floating iea_15MW turbine. and any suggestions to where i can setup the windfarm layout of 50 turbines?

Thank you so much in advanced
